packages/adapters/src/adapters/helpers/codeMeaningEquals.ts

```diff
@@ -0,0 +1,9 @@
+const codeMeaningEquals = codeMeaningName => {
```

Comment:
Could you add a docs here - this isn't a strict comparison of two values, but generates a function that compares the originally provided value with a content item.